Documentation/development-flow.md

```diff
@@ -52,6 +52,47 @@ git remote -v
 ```
 Now you should have at least `origin` and `upstream` remotes. You can also add other remotes to collaborate with other contributors.
 
+## Layout
+
+A source code layout is shown below, annotated with comments about the use of each important directory:
+
+```console
```

Comment:
Why not just display it as a (html) list instead of a code block?

Reply:
Okay, updating